docs/debugging_buildfailures.md

```diff
@@ -0,0 +1,14 @@
+# Capturing binlogs
+Capturing and providing binlog files can help with debugging build and packaging issues. In order to collect binlogs, please follow these steps:
+
+1. Download the **[VS Project System Tools extension](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.ProjectSystemTools)**
+2. Set the **Build Log File** verbosity to `Diagnostics`: `Tools->Options->Projects and Solutions->MSBuild project build log file verbosity`:<br/>
+![Screenshot of Build and run options](./images/binlogs_buildandrunoptions.png)
+
+3. Go to View->Other Windows->Build Logging:<br/>
+![Screenshot of Build Logging menu item](./images/binlogs_buildlogging_menuitem.png)
```

Comment:
As more and more images get created I think we should start creating subfolders for the image directory and thus sort images that way for easier maintenance and organization.

Reply:
Great idea! Moved them into a binlog folder for now.